mlu_op.h

```diff
@@ -14523,6 +14523,112 @@ mluOpLgamma(mluOpHandle_t handle,
             const mluOpTensorDescriptor_t y_desc,
             void *y);
 
+/*!
+ * @brief Computes the Cholesky factorization of a Hermitian positive-definite matrix.
+ *
+ * @param[in] handle
+ *   Handle to a Cambricon MLUOP context that is used to manage MLU devices and queues in the
+ *   deformable convolution operation. For detailed information, see ::mluOpHandle_t.
+ *
+ * @param[in] input_desc
+ *  The descriptor of the input matrix to factorise, it is an n×n Hermitian matrix,
+ *  only the lower or upper part is meaningful.
+ *
+ * @param[in] d_input
+ * Pointer to the MLU memory that stores the input matrix.
+ *
+ * @param[in] output_desc
+ * The descriptor of the result matrix, it is an n×n lower triangular matrix or an upper triangular matrix.
+ *
+ * @param[out] d_output
+ * Pointer to the MLU memory that stores the result matrix.
+ *
+ * @param[in] upper
+ * upper indicates which part of the matrix is used.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_EXECUTION_FAILED
+ *
+ * @par Data Type
+ * - The supported combinations of data types are shown below:
+ *   - float(\b d_input) - float(\b d_output)
+ *
+ * @par Data Layout
+ * - The data layout of d_input should be \p MLUOP_LAYOUT_ARRAY.
+ * - The data layout of d_output should be \p MLUOP_LAYOUT_ARRAY.
+ *
+ * @par Scale Limitation
+ * - The dimension of input/output tensor must be 2 or 3.
+ * - The shape of output should be equal to input shape.
+ * - When the dimension of tensor is 2, the shape[0] and shape[1] should be equal.
+ * - When the dimension of tensor is 3, the shape[1] and shape[2] should be equal.
```

Comment:
```suggestion
 * - When the dimension of the tensor is 2, the shape[0] and shape[1] should be equal.
 * - When the dimension of the tensor is 3, the shape[1] and shape[2] should be equal.
```
这两条可以归属到第一条的两种情况吗？这样？
- The dimension of input/output tensor must be 2 or 3.
  - When the dimension of the tensor is 2, the shape[0] and shape[1] should be equal.
  - When the dimension of the tensor is 3, the shape[1] and shape[2] should be equal.